framework/tests/aliasing/UniqueConstructorTest.java

```diff
@@ -0,0 +1,24 @@
+import org.checkerframework.common.aliasing.qual.Unique;
+
+public class UniqueConstructorTest {
+    // No need for @SuppressWarnings()
+    @Unique UniqueConstructorTest() {
+        // Doesn't raise unique.leaked error since it's a top parent class (@Unique Object subclass
```

Comment:
What does `top parent class` mean?

Reply:
The Parent class extends UniqueConstructorTest and the Child class extends the parent class. Hence, I referred to the UniqueConstructorTest class as the "top parent class" (not related to the Parent class in the code) in the inheritance hierarchy. Since, it explicitly doesn't extend any class, it extends java.lang.Object class by default.